### PR TITLE
Remove ERP verbs

### PR DESCRIPTION
**Pull request by rhicora**
Rationale for this change:

- The use of the verbs encourages LRP gameplay
- The descriptive writing surrounding the verbs is low quality and repetitive
- The use of the verbs regularly replaces high-quality emotes
- The verbs have no notion of OOC consent, which routinely violates Rule 1
- The verbs were introduced solo by the previous host and did not receive code review

### CODE REVIEW

**Comment by Monika4** [comment]
Death of ERP verb is here and it is queer

**Comment by JarlElbow** [comment]
So, this is by far the weirdest PR I've ever commented on, but oh well.

I disagree with this change for a pretty simple reason. Sure, removing those verbs would make people have to be creative and actually write out their RP, however it'd create a bigger problem. We already have people complaining about others spending a large portion of the round ERPing somewhere, such as in the vault, without naming any names. So how much worse would it be if instead of just pressing a button a couple dozen times, they actually had to type out all of that? 

If you've been fortunate enough to never have been on citadel, you know that people literally spend upwards of 2 hours in a round doing nothing but ERP. This change would probably do the same thing,  and make all these problems even worse. 

TLDR: If someone wants to type out everything, they can. If they don't they don't have to. Give them a choice, instead of limiting their options and creating even bigger issues.

**Comment by DorfGuy1** [comment]
This is a bad Idea for the following reasons 
it removes features
will probably result in less people coming to the server, which results in a feedback loop.
no more legion/raider rape train shenanigans. 
To counter the "we are not removing erp only verbs meme" removing verbs would make things take forever, which would essentially remove the legion/raider rape shenanigans in their current form
the current form is hilarious.

**Comment by companoo** [comment]
i dont like erp but people do

should probably update it rather then remove it

that or do an opinion poll or something

**Comment by EndaruI** [comment]
yeah this inspired me to erp with me emote to creep guys the fuck out

**Comment by EvenInDeathIStillServe** [comment]
- The use of the verbs encourages LRP gameplay
How so? The only possible example of this is spam-clicking the ERP buttons, or memeraping someone's corpse after killing them, the first could be blocked with a cooldown and the second should damage you with toxins or give you diseases.

- The descriptive writing surrounding the verbs is low quality and repetitive
Make the actions have more interesting and varied descriptions, instead of removing the mechanics entirely.

- The use of the verbs regularly replaces high-quality emotes
A baseless assumption. How did you come to this conclusion? People who like to write "high-quality emotes" will do so no matter if ERP verbs are present or not. 

- The verbs have no notion of OOC consent, which routinely violates Rule 1
By this logic, you should be able to avoid all conflict by not consenting to it. If you need to consent to rape, you logically would have to consent to being murdered. You kind of agree that anything can happen to your character, including rape, when you join the server after confirming that you are of legal age.

- The verbs were introduced solo by the previous host and did not receive code review
Doesn't mean you have to remove them outright.

Having ERP as an actual gameplay mechanic is beneficial for player immersion by adding audio (sound effects) and visual (cum decal) elements to an activity. The presence of sound effects means you have to find a secluded location to perform your deeds, unless you want nearby people to hear and intentionally barge in on you, and the cum decal means you also have to worry about cleaning up after the act if you don't want people to know what you were up to.

ERP verbs allow easier access for inexperienced roleplayers to create interesting interactions with other players. Believe it or not, some people play on this server just to have sex. If you take away the ERP tools from the common player, sex as a whole will probably disappear. The paragraph ERPers will either ERP with eachother and ignore all other players, or they will leave.

ERP verbs are not a problem. There is no need for this removal.

**Comment by imw-Oculus** [comment]
I think changing would be more in order than removing. Though, I'm not sure what changes you could make.

**Comment by Cadwell-Regis** [comment]
@EvenInDeathIStillServe 
There is... really obvious bias in how you are talking about the verbs, like as if they promote something great out of the server and attract wonderful people - who make an effort to entertain the people around them. Even though you have people fingering deathclaws, corpses, tying people up and raping them - using the verbs to troll half the server, breaking immersion as they drag around a cadaver and bootgrind it across the map. I mean either you are knee deep in confirmation bias or I am, because I can't see how the ERP Verbs help with immersion given how often they are used to do the complete opposite. 

If that kind of person is going to leave the server, because they can't do that shit anymore - I don't think I'd very much care. If you read up on Reddit about this server, a lot of potential roleplayers have avoided this server purely because of this stuff, there are a lot of SS13 servers that don't have ERP at all. They get along fine, and they don't give a shit if someone is going to throw a tantrum because they can't tie someone up and force them into an uncomfortable situation that makes them end up leaving the server forever.

Rule 1 from what I remember, the new one, is purposed towards storytelling - the idea behind it was that your goal shouldn't be to go around ruining people's rounds for your own benefit. That there is an obligation from you to contribute to everyone's fun, not just your own. 

Removing this trash, can help F013 be a whole lot better then it is, I stopped playing a while ago because of the constant RDM and ERP verbs myself, but I'd still like to keep contributing in the hopes it might be better one day. So it's already in a way, made me stop being a player myself and I know from having met decent people, that the best kinds of roleplayers, people whose sole focus is entertaining the people around them, would not play because of shit like this.

A direction is clearly being taken here to make something better then what we have, not to compromise or give in, to pander both to assholes and good people out of some pointless fear of the assholes leaving. Because from what I've seen so far, there are no plans whatsoever to help anyone who wants to ruin the game for the rest of us, there are only measures being taken to benefit people who make this place fun for others.

**Comment by angrycupholder** [comment]
I don't believe that removing the verbs is the way out. The prose should certainly be improved but removing it would be negative to the serb pop and culture, since you know, sex shenanigans has become a part of BD.

**Comment by ShitbagStaton** [comment]
if you want your server to forever be known as that place that people go to erp, then congratulations on becoming the next citadel. you have the ability to attract people actually interested in rp, and put them in a fallout universe where we can play out characters in this shitty wasteland. instead, the people coming to join are people who wear dresses and walk up and down the roads "avoiding" being raped, you get erp dens every round and people just erp for the entire round and contribute absolutely nothing. u can erp verb spam rape someone in about 10 seconds and it doesnt even have any effect what so ever because its so common and annoying its just an eye-roll now. if you remove erp verbs you will lose population, yes, you will lose people who lowrp and fuck corpses. in time it will be replaced by people who actually want to have a purpose other then just fucking every ghoul corpse they see depending on how you present the server afterwards. you have people ghosting from their characters and just giving up on the round because a retard with a lime green afro drags them off and verb spam rapes them and they want no part of it. your tailoring your population in the worse way possible for a "MRP" server when it is not at all even close. 

erp is going to happen regardless because there is a me function. erpers will live on, the only people seriously wanting to keep this shit is people wanting to meme out of it

i.e in the enclave event a guy found a FEV experiment and the first thing he did was mouth fuck it

if you do not remove them then at least update them, make it take time for them to do and make it actually significant in someway, and make it so if i dont want to be involved in that stupid shit i have a way of doing it without ghosting and losing my round.

**Comment by Monika4** [comment]
memes in part of this will kind of be gone and it will sad to see them but im more talking from a point that verbs are going to get removed which most likely they are because it looks like it is but namely we should look at how they are used over a period of time or something along those lines because rape ERP is kind of funny the first time like "oh I got raped by a man called rape man" like its clear that some times it can be funny to spam the living shit out of these verbs as in the case of Foot and mouth when the round is drawing to a close. and sometimes it is used in place of trying to make some faction as bad as in place of legion and raiders and it is kind of gameplay element in a sense allow better Player to Player interaction but it can be used as a shit meme which most of the time it is 


in my closing statement I shall leave with this nothing shall change in a code point but however it be player base shall be divided on this issue for ages to come either to "increase the RP of ERP" or to "have verbs for better and more viable RP in ERP"

**Comment by k-k-karma** [comment]
lmao fucking niggers arguing over erp

**Comment by Kimochiiiiiiiiiiiiii** [comment]
removing is no good, muh raider rp, they need to erp when needed but the playerbase thinks most raiders just go for memes and ERP for the laughs. Removing it fully Will do no good, oof. moderate it.

**Comment by pireamaineach** [comment]
Bring it back.

**Comment by waterwater1212** [comment]
Bring it back.
